README.md

```diff
@@ -130,3 +130,32 @@ VITE_ACCRUALS_API_URL_PROXY="http://localhost:9091/"
 ## Troubleshooting
 
 If you’re using an AMD (M1,M2) chip mac, you might encounter problems starting the keycloak docker container. We’ve had some success building a new image locally, as per the instructions in this github thread: https://github.com/docker/for-mac/issues/5310#issuecomment-877653653
+
+## Running UI in LocalDev Environment
+
+### Step 1
+
+Download Callisto LocalDev repository from https://github.com/UKHomeOffice/callisto-localdev
+
+Run its services with command : `docker compose up -d`
+
+### Step 2
+
+Stop (manually) service `web` (most likely the name will be `web-1`)
```

Comment:
We should change this to use the docker compose method
`docker compose stop web`

Reply:
sure, let's use docker compose for that
Fixed.